MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/controller/ScreeningsController.java

```diff
@@ -0,0 +1,41 @@
+package p.lodz.pl.multiplexreservationsystem.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import p.lodz.pl.multiplexreservationsystem.service.SeatsService;
+import p.lodz.pl.multiplexreservationsystem.service.dto.RoomInfoDto;
+import p.lodz.pl.multiplexreservationsystem.service.dto.ScreeningsDto;
+import p.lodz.pl.multiplexreservationsystem.service.ScreeningsService;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class ScreeningsController {
+
+  private final ScreeningsService screeningsService;
+  private final SeatsService seatsService;
+
+  @GetMapping("/screenings")
+  public List<ScreeningsDto> getScreeningsWithMovies(@RequestParam(required = false) int page) {
```

Comment:
1. required=false i mały int, jak to się zachowa?
2. To jest endpoint, który zwraca wszystkie filmy świata wraz ze wszystkimi wyświetleniami?

Reply:
Rzeczywiście! int to typ prymitywny i nie może być nullem.

Reply:
Ahh rzeczywiście, obecnie zostałyby wyciągnięte dane archiwalne, lub te 15 minut przed rozpoczęciem seansu...